docs/protocol/papers/blockchain-programs.md

```diff
@@ -453,16 +454,16 @@ While most state should be tracked locally in the program-level arguments for a
 
 First, one needs to create an asset for which only one unit can ever be issued. This requires some understanding of how the Chain Protocol handles issuances. Unique issuance — ensuring that issuances cannot be replayed — is a challenging problem that is outside the scope of this paper. The Chain Protocol’s solution is that each issuance input has a nonce that, when combined with the transaction’s `mintime`, `maxtime`, and asset ID, must be unique throughout the blockchain’s history. As a result, an issuance *program* can ensure that it is only used once by committing to a specific nonce, transaction mintime, and transaction maxtime:
 
-    program SinglyIssuableAssetSingletonToken(nonce, mintime, maxtime, amount, lockProgram) {
-    	path issue(outputIndex) {
+    program SinglyIssuableAssetSingletonToken(nonce: String, mintime: Time, maxtime: Time, amount: Number, lockProgram: Program) {
```

Comment:
Mintime and maxtime are the words used throughout the protocol; maybe worth discussing (though out of scope of this PR)